deploy.sh

```diff
@@ -0,0 +1,96 @@
+#!/bin/bash
+
+# Deploy a version of pdc from a repo based on the contents of a file.
+#
+# The file can be written by an unprivileged user. The deployment user
+# running this script can be a more-privileged user. The file contains
+# a tag. The context for which project and file to use is from
+# variables in a .env file.
+#
+# For example, if the file contains "v1", the .env file may contain a
+# variable with the url prefix such as "https://pdc/deploy/" and the
+# .env also includes a variable with the file name "compose.yml", then
+# the compose script to get/copy will be something like the following:
+# https://pdc/deploy/v1/compose.yml
+#
+# After getting the compose script, "down" the old one (if there) and
+# "up" the new one.
+#
+# Following successful processing of the tag file, this script deletes
+# it.
+#
+# Requires docker-compose or docker with the compose plugin.
+# Requires a .env file with two variables, see below.
+
+DOCKER_COMPOSE_COMMAND="docker-compose"
```

Comment:
This is an annoying variable to have, but in my local machine I have the compose plugin (e.g. `docker compose`) whereas other machines have the (older-style) `docker-compose`.